context/app/api/vitessce.py

```diff
@@ -320,8 +334,36 @@ def _build_vitessce_conf(self):
             found_images = _get_matches(file_paths_found, files[0]["rel_path"])
             # Do not show IMS images that are in the "/separate/" folder.
             no_ims_separate_images = _exclude_matches(found_images, r"/separate/")
-            layer = self._build_multi_file_image_layer_conf(no_ims_separate_images)
-            conf["layers"] = [layer]
+            # File paths are like path/to/HybCycle_N/MMStack_PosM.ome.tif and we want
+            # each conf to be all images across a given PosM, each image named HybCycle_N.
+            # This is unique to seqFish.
+            if "seqFish" in self.entity["data_types"]:
+                is_valid_directory = all([re.match(SEQFISH_PATH, file) for file in found_images])
```

Comment:
```suggestion
                is_valid_directory = all([re.fullmatch(SEQFISH_PATH, file) for file in found_images])
```
`match` allows unmatched characters at the end.